### ISSUE TITLE
version of mmrotate mismatch

### ISSUE THREAD

**Issue by yuhua666** [ViTAE-Transformer/Remote-Sensing-RVSA#53]
What version of mmrotate are you using? For example, I found that OBBShared2FCBBoxHead is missing from my mmrotate.

**Comment by DotWang** [comment]
@yuhua666 The original RVSA detection (this repo) is implemented on OBBDetection. As I know, the early mmrotate, may be a version of 0.X, was developed based on it.

We have implemented the RVSA series model using OpenMMLAB packegs in a new repo, see MTP: https://github.com/ViTAE-Transformer/MTP

**Comment by yuhua666** [comment]
Thank you for your reply. May I also ask how many and what type of GPU did your MTP project use for downstream tasks (DIOR-R dataset or other remote sensing datasets)? Approximately how much video memory is needed?

**Comment by DotWang** [comment]
@yuhua666 When fine-tuning on DIOR-R using MTP models, we use the GPU has 16G memory, where each batch is on a GPU. So 3090 or 4090 is enough.

**Comment by yuhua666** [comment]
Thank you for your reply. I will work hard to learn from your excellent project. Also, I have a small question. Can I train with a 4090 graphics card on other datasets? This doubt comes from GPU overflow when I set one batch size during RVSA training.

**Comment by DotWang** [comment]
@yuhua666 Do you mean using only one 4090 GPU to train, with a batch size of 2 or 4? The RVSA-B equipped with the gradient checkpointing may be available, but I'm not sure, you can try it out.

**Comment by yuhua666** [comment]
I can accept using multiple 4090 GPUs for training, but I don't have a separate graphics card with large memory, such as A100. When the batch size is set to 1, can a single 4090 accommodate it, such as training the DOTAv1.0 dataset.

**Comment by DotWang** [comment]
@yuhua666 theoretically it's ok, however, in some neck or detection heads, the batch normalization layer may not allow batch size=1

**Comment by yuhua666** [comment]
I noticed that the file organization of the MTP project is different from mmrotate. Can this project directly train remote sensing object detection independently, or does it need to be organized into the structure of mmrotate?

**Comment by DotWang** [comment]
@yuhua666 you should move the files in this path: https://github.com/ViTAE-Transformer/MTP/tree/main/RS_Tasks_Finetune/Rotated_Detection/mmrotate1.x in your installed MMRotate, see MTP's readme
